src/client/views/custom-functions-dashboard.html

```diff
@@ -33,6 +33,7 @@
 
 
 	<script>
+		window.playground_host_ready = true; // =FIXME OMG don't check in
```

Comment:
Please don't check this one in.

Reply:
actually, you could probably replace this whole script tag with:

    Office.onReady(() => window.playground_host_ready = true;)

And then it should work fine both inside an add-in and for your web-based testing